Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -1016,21 +997,66 @@ public void OrderTxsForEvaluation()
                 "0xfcbfa4977B2Fc7A608E4Bd2F6F0D6b27C0a4cd13",
                 "0xB0ea0018Ab647418FA81c384194C9167e6A3C925",
             };
-            string[] orderedAddresses =
-            {
-                "0x921Ba81C0be280C8A2faed79E14aD2a098874759",
-                "0x1d2B31bF9A2CA71051f8c66E1C783Ae70EF32798",
-                "0xB0ea0018Ab647418FA81c384194C9167e6A3C925",
-                "0xfcbfa4977B2Fc7A608E4Bd2F6F0D6b27C0a4cd13",
-                "0xc2A86014073D662a4a9bFCF9CB54263dfa4F5cBc",
-            };
-
+            // Sanity check.
             Assert.True(originalAddresses.SequenceEqual(
                 signers.Select(signer => signer.ToAddress().ToString())));
-            Assert.True(orderedAddresses.SequenceEqual(
-                orderedTxs
-                    .Where((tx, i) => i % numTxsPerSigner == 0)
-                    .Select(tx => tx.Signer.ToString())));
+
+            // Make sure to cover every case *and* the currently running protocol does not break.
+            int[] protocolVersions = { 0, 3, BlockMetadata.CurrentProtocolVersion };
+            foreach (int protocolVersion in protocolVersions)
```

Comment:
Rather than placing a loop in a test case, it'd be better to [parametrize the test case using `[Theory]`][1] attribute.

[1]: https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/

Reply:
Yeah... this was a long time ago before I knew about the `[Theory]` attribute. 🙃
See https://github.com/planetarium/libplanet/pull/1518/commits/fa8fc9627adc746785660d6be4e942503a6960df.